src/context/mod.rs

```diff
@@ -429,7 +815,49 @@ impl<'ctx, CTX: private::SealedContext<'ctx> + private::SealedCanReadState> Cont
 {
 }
 
-/// Trait exposing Restate K/V write functionalities.
+/// # State
```

Comment:
This has the same title of the previous one, let's use a different titles, e.g. "Reading state" and "Writing state"